calculator-project/src/Components/Solution.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+const Solution = ({result}) => {
+
+  return <p className="result">{Number(result).toLocaleString() || 0}</p>;
```

Comment:
I see you use Number here, but parseInt other places. I would stay consistent with one across your code